src/excel_adapter/internals/excel_reader.go

```diff
@@ -26,8 +26,8 @@ func parseSheets(file *excelize.File) (models.Sheet, map[string]models.Sheet) {
 	sheetMap := file.GetSheetMap()
 	for _, name := range sheetMap {
 		cols := getSheetCols(file, name)
-		if strings.HasPrefix(name, BOARD_SHEET_PREFIX) {
-			boardSheets[strings.TrimPrefix(name, BOARD_SHEET_PREFIX)] = parseSheet(name, cols)
+		if !strings.HasPrefix(name, GLOBAL_SHEET_PREFIX) {
+			boardSheets[strings.TrimPrefix(name, GLOBAL_SHEET_PREFIX)] = parseSheet(name, cols)
```

Comment:
Is this correct? You are removing the `GLOBAL_SHEET_PREFIX`, maybe you intended to remove the `BOARD_SHEET_PREFIX`

Reply:
This works better, there is only one global sheet and the rest are all boards, this way the names on the excel adapter for boards are just the board name and then the sheet with the global info is the one that has a prefix